### ISSUE TITLE
CLI "mangles"/mixes output

### ISSUE THREAD

**Issue by janpio** [ionic-team/capacitor#267]
(Is "mangle" actually a proper English word?)

Seems like the CLI is sometmes mixing output together, probably because it is executing stuff at the same time:

```
C:\Projects\Capacitor\pluginDemos (HEAD detached at 0c5cc7b -> origin) (capacitor-app@1.0.0)
λ npx cap update
C:\Projects\Capacitor\pluginDemos\node_modules\@capacitor\cli\bin\capacitor
√ update web in 43.28μp
√ Fetching plugins in 85.27ms
Found 1 Cordova plugin(s):
  cordova-plugin-network-information
\ update androidFound 0 Capacitor plugin(s):

√ update android in 226.26ms
``` 

`\ update android`  seems to be mixed in there.

```
C:\Projects\Capacitor\pluginDemos (HEAD detached at 0c5cc7b -> origin) (capacitor-app@1.0.0)
λ npx cap copy
C:\Projects\Capacitor\pluginDemos\node_modules\@capacitor\cli\bin\capacitor
√ Copying web assets from www to android\app\src\main\assets\public in 57.09ms
√ Copying native bridge in 7.37ms
Found 1 Cordova plugin(s):
  cordova-plugin-network-information
√ copy in 204.89ms
- copyCopying web!
√ Copying capacitor.js to web dir in 7.50ms
√ copy in 25.07ms
```

Here `- copy` looks out of place. The "spinner" also seems to have stopped to early and not finished to `√`. 

Anyone else having this problem?

(Note that I am on Windows 10, Cmder is my command line right now.)

PS: More examples added as I encounter them:

```
λ npx cap add android
C:\Projects\Capacitor\npmInitTest\node_modules\@capacitor\cli\bin\capacitor
√ Adding native android project in: C:\Projects\Capacitor\npmInitTest\android in 97.61ms
- Syncing GradleCreating local properties C:\Users\Jan\Library\Android\sdk
√ Syncing Gradle in 1.07ms
√ add in 108.36ms
√ Copying web assets from www to android\app\src\main\assets\public in 8.64ms
√ Copying native bridge in 2.24ms
√ copy in 30.64ms
√ Fetching plugins in 2.30ms
Found 0 Capacitor plugin(s):

√ update android in 24.52ms
```

**Comment by mlynch** [comment]
Probably related to #181

**Comment by mlynch** [comment]
Closed in 41b2bf0
